lib/views/widgets/divider/custom_vertical_divider.dart

```diff
@@ -0,0 +1,13 @@
+
+import 'package:flutter/material.dart';
+import 'package:rich_text_composer/views/commons/colors.dart';
+
+class CustomVerticalDivider extends StatelessWidget {
```

Comment:
Can `VerticalDivider` widget replace this?

Reply:
No, we need custom it. VerticalDivider is not rendering as we expected